universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts

```diff
@@ -13,16 +13,25 @@ export const getMinimalAmountForFiatProvider = async (
   switch (paymentMethod) {
     case TopUpProvider.RAMP: {
       const currencyPriceInEth = await tokenPricesService.getTokenPriceInEth(currencyDetails);
-      const providerMinimalAmountInFiat = '1';
+      const providerMinimalAmountInFiat = ['0.6', '2'];
       const etherPriceInGBP = (await tokenPricesService.getEtherPriceInCurrency('GBP')).toString();
-      const providerMinimalAmount = getPriceInEther(providerMinimalAmountInFiat, etherPriceInGBP);
+      const providerMinimalAmount = getPriceInEther(providerMinimalAmountInFiat[0], etherPriceInGBP);
```

Comment:
Could you tell me what currency this minimal amount is? Because I heard EUR but here is GBP. So I can see here inconsistency.